src/main/java/ch/njol/skript/bukkitutil/BukkitUnsafe.java

```diff
@@ -60,10 +60,7 @@
 	private static final boolean knownNullPtr = !Skript.isRunningMinecraft(1, 11);
 	
 	static {
-		UnsafeValues values = Bukkit.getUnsafe();
-		if (values == null)
-			throw new Error("UnsafeValues not available");
-		unsafe = values;
+		unsafe = Bukkit.getUnsafe();
```

Comment:
why is it safe to remove the exception that may be thrown here? it's not guaranteed that this API exists

Reply:
Wait, when would this API not exist? If the user is running some non-Bukkit software?

Reply:
It's in the class docs: "Their existence and behavior is not guaranteed across future versions."

Reply:
I re-did the code section for that, I think it should solve this?